packages/@o3r/third-party/collection.json

```diff
@@ -6,6 +6,12 @@
       "factory": "./schematics/ng-add/index#ngAdd",
       "schema": "./schematics/ng-add/schema.json",
       "aliases": ["install", "i"]
+    },
+    "add-iframe-component": {
```

Comment:
`iframe-to-component` ? to stay aligned with the others schematics applied to components (loc-to-component, context-to-component etc)